packages/financial-templates-lib/src/price-feed/DefiPulseTotalTVLPriceFeed.js

```diff
@@ -0,0 +1,164 @@
+const { PriceFeedInterface } = require("./PriceFeedInterface");
+const { parseFixed } = require("@ethersproject/bignumber");
+
+// An implementation of PriceFeedInterface that uses DefiPulse Data api to retrieve prices.
+class DEFI_PULSE_TOTAL_TVL_PriceFeed extends PriceFeedInterface {
```

Comment:
nit: while the price feed is indeed called DEFI_PULSE_TOTAL_TVL This class should not be named like this. Ideally, it should be called DefiPulsePriceFeed I believe.

Reply:
Using DefiPulseTotalPriceFeed so I can differentiate between it and the SUSHI/UNI TVL price feed in the future